src/styles/blocks/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: var(--background-color);
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  color: var(--main-color);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.card__code {
+  color: var(--secondary-color);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  align-items: flex-end;
+}
+
+.card__reviews-count {
+  font-size: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  font-weight: 400;
+}
+
+.card__price-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-name {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: var(--secondary-color);
+}
+
+.card__price-price {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card__buy-button {
+  box-sizing: border-box;
```

Comment:
you already added box-sizing in `*` selector, delete here

Reply:
++